src/main/java/legalthings/lto_api/lto/core/Account.java

```diff
@@ -0,0 +1,260 @@
+package legalthings.lto_api.lto.core;
```

Comment:
change the namespace according to the package name so `one.legalthings.sdk`

Reply:
OK